include/alpaka/core/Positioning.hpp

```diff
@@ -24,6 +24,17 @@
 namespace alpaka
 {
     //-----------------------------------------------------------------------------
+    //! Defines the parallelism hierarchy levels of Alpaka
+    //-----------------------------------------------------------------------------
+    namespace hierarchy
+    {
+        struct Grids{};
```

Comment:
Could you please add the `//#########...` comment above the struct definitions?
I think you could remove the braces as they are optional.